datastructs/ring.h

```diff
@@ -9,20 +19,87 @@
 
 struct ring{
         /* Fill this in */
+        int size;
+        int pop;
+	int push;
+        ptr_t *items;
+}
+
+int isEmpty(struct ring *rb)
```

Comment:
//Got you, thank you so much for explaining over the different in name tags !

-int isEmpty(struct ring *rb)
+int ring_is_empty(struct ring *rb)